api/module.mk

```diff
@@ -28,22 +31,31 @@ TEST_SCM_SRC       = $(shell $(FIND) $(TEST_DIR) -type f -name "*.$(CLJ)")
 JAR          := $(DERIVED_TARGET_DIR)/wfl-$(WFL_VERSION).jar
 JAR_LINK     := $(DERIVED_TARGET_DIR)/wfl.jar
 
-$(PREBUILD): $(MODULE_DIR)/wfl
+$(PREBUILD): $(SCM_SRC) $(SCM_RESOURCES)
 	@$(MKDIR) $(DERIVED_RESOURCES_DIR) $(DERIVED_SRC_DIR)
-	$(BOOT) prebuild
+	$(CLOJURE) -X wfl.boot/prebuild
 	@$(TOUCH) $@
 
-$(MODULE_DIR)/wfl: $(BOOT_PROJECT)
-	$(LN) $(BOOT_PROJECT) $@
+$(POM_IN): $(PREBUILD) $(CLOJURE_PROJECT)
```

Comment:
Not sure if you need to depend on PREBUILD since you already have BUILD -> POM_OUT -> POM_IN? and BUILD needs PREBUILD?

Reply:
The pom needs the calculated version -- at least in principle.

Reply:
Actually, I guess only POM_OUT needs the version, since POM_IN ignores it.

Reply:
But now POM_OUT has more than one direct dependency. :sigh: